login/index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://sad-haibt-42112c.netlify.app/styles/main.css"
+    />
+    <link rel="stylesheet" href="/login/login.css" />
+    <title>Login | Sanguine Store</title>
+  </head>
+  <body>
+    <nav class="navigation">
+      <div class="nav-logo">SANGUINE store</div>
+      <label for="search" class="text-size-sm">
+        <input
+          type="search"
+          class="search-box padding-md border-rad-md"
+          name="search"
+          id="search"
+          placeholder="What are you looking for?"
+        />
+        <a href="" class="options text-dec-none">Search</a>
+      </label>
+      <ul class="list-style-none list-inline">
+        <li>
+          <a class="options link" href="">profile</a>
+        </li>
+        <li>
+          <a class="options link" href="">wishlist</a>
+        </li>
+        <li>
+          <a class="options link" href="">cart</a>
+        </li>
+      </ul>
+    </nav>
+    <section class="login-form">
+      <div class="input-container">
+        <div class="card card-vertical border-rad-md box-shadow margin-lg">
+          <h3 class="h3 text-wt-bold">Login</h3>
+          <label for="input-text">E-mail or username</label>
+          <input
+            type="text"
+            name="input-text"
+            class="txt-box"
+            id="input-text"
+            placeholder="E-mail or username"
+            required
+          />
+
+          <label for="input-pwd">Password</label>
+          <input
+            type="password"
+            name="input-pwd"
+            class="pwd-box"
+            id="input-pwd"
+            placeholder="********"
+          />
+
+          <button class="btn btn-primary" type="submit">Login</button>
+          <span class="text-size-sm"
+            >Don't have an account?
+            <a class="link text-style-underline" href="">Sign up</a></span
```

Comment:
UI suggestion: You can change the color of sign up text so that it matches with your UI.

Reply:
Yes, will try to do this, thanks.